requirements.txt

```diff
@@ -1 +1 @@
-Theano==0.7
+git+https://github.com/Theano/Theano.git@4dacb9e7d04d842d8ffe2c948dbc90722ae990d4#egg=Theano
```

Comment:
I really don't think that makes sense. Theano went for over a year between releases, who knows when the next one will be. 

We should encourage people to use the git version, if we specify a commit in `requirements.txt`, it will be installed when people `pip install Lasagne`.